ui/tests/helpers/openapi/auth-model-attributes.js

```diff
@@ -605,6 +605,14 @@ const kubernetes = {
       label: 'Token Reviewer JWT',
       type: 'string',
     },
+    useAnnotationsAsAliasMetadata: {
+      editType: 'boolean',
+      fieldGroup: 'default',
```

Comment:
It looks like the fieldGroup above is `Kubernetes Options`. I can't remember how this openAPI test is setup, does this need to match above?

Reply:
Thanks for confirming and following up!